src/IdBrokerClient.php

```diff
@@ -8,6 +8,26 @@
 
 /**
  * IdP ID Broker API client implemented with Guzzle.
+ * @method Result authenticateInternal(string[] $parameters) Authenticate user.
```

Comment:
I don't think we need/want to call out the `Internal` methods as they are the internal Guzzle methods that we don't expect a user to call directly. That is why we suffixed them with `Internal` and exposed public methods without that suffix for general use.